shared-components/tl-tournament-card.js

```diff
@@ -95,50 +37,133 @@ export class TournamentCard extends LitElement {
-                    line-height: var(--paragraph-medium-line-height);
-                    font-weight: var(--paragraph-medium-font-weight);
+                font-size: var(--paragraph-medium-font-size);
+                line-height: var(--paragraph-medium-line-height);
+                font-weight: var(--paragraph-medium-font-weight);
                 }
 
                 mwc-button[outlined] {
-                    --mdc-shape-small: 2rem;
-                    --mdc-theme-on-primary: white;
+                --mdc-shape-small: 2rem;
+                --mdc-theme-on-primary: white;
 
-                    --mdc-button-disabled-fill-color: #7644b5;
-                    --mdc-button-disabled-outline-color: var(--mdc-theme-primary);
+                --mdc-button-disabled-fill-color: #7644b5;
+                --mdc-button-disabled-outline-color: var(--mdc-theme-primary);
 
-                    filter: drop-shadow(0px 1px 23px #b973e0);
-                    --mdc-theme-primary: null;
-                    --mdc-button-outline-color: #c078e4;
+                filter: drop-shadow(0px 1px 23px #b973e0);
+                --mdc-theme-primary: null;
+                --mdc-button-outline-color: #c078e4;
                 }
 
                 .button-label-large {
-                    --mdc-typography-button-font-family: var(--font-family);
-                    --mdc-typography-button-font-size: var(--button-large-label-font-size);
-                    --mdc-typography-button-line-heightt: var(--button-large-label-line-height);
-                    --mdc-typography-button-font-weight: var(--button-large-label-font-weight);
-                    --mdc-typography-button-text-transform: none;
-                    --mdc-typography-button-letter-spacing: none;
+                --mdc-typography-button-font-family: var(--font-family);
+                --mdc-typography-button-font-size: var(
+                    --button-large-label-font-size
+                );
+                --mdc-typography-button-line-heightt: var(
+                    --button-large-label-line-height
+                );
+                --mdc-typography-button-font-weight: var(
+                    --button-large-label-font-weight
+                );
+                --mdc-typography-button-text-transform: none;
+                --mdc-typography-button-letter-spacing: none;
+                }
+
+                .card-header {
+                position: relative;
+                }
+
+                .card-image {
+                width: 100%;
+                height: 20rem;
+                border: 2px solid rgba(34, 45, 56, 0.5);
+                border-radius: 2rem;
+                object-fit: cover;
+                }
+
+                .card-active-date {
+                position: absolute;
+                top: 0;
+                right: 2.5rem;
+                transform: translate(0, -50%);
+                border-radius: 255px;
```

Comment:
From my experience, it is a great way to make a tag look completely rounded if we don't know that tag's height.
For example: if the height is 30px, we can set border-radius 15px or 255px.  it will look like each other definitely.